connectors/src/connectors/github/lib/github_api.ts

```diff
@@ -568,37 +568,99 @@ export async function getOctokit(connectionId: string): Promise<Octokit> {
 // Repository processing
 
 const EXTENSION_WHITELIST = [
+  // Programming Languages - General Purpose
   ".js",
   ".ts",
   ".tsx",
   ".jsx",
-  ".rb",
   ".py",
+  ".rb",
   ".rs",
   ".go",
   ".swift",
-  ".css",
-  ".html",
-  ".htm",
-  ".less",
-  ".sass",
-  ".scss",
-  ".php",
   ".java",
-  ".yaml",
-  ".yml",
-  ".md",
   ".c",
   ".h",
   ".cc",
   ".cpp",
   ".hpp",
+  ".php",
+
+  // .NET Ecosystem
+  ".cs",
+  ".csproj", // XML-based
+  ".sln", // Text-based solution file
+  ".cshtml", // Razor template
+  ".razor", // Razor component
+  ".resx", // XML-based resource
+  ".vb", // Visual Basic
+  ".fs", // F#
+  ".fsproj", // XML-based F# project
+  ".props", // MSBuild properties (XML)
+  ".targets", // MSBuild targets (XML)
+  ".nuspec", // NuGet specification (XML)
+
+  // Web Technologies
+  ".html",
+  ".htm",
+  ".css",
+  ".scss",
+  ".sass",
+  ".less",
+
+  // Data & Configuration
+  ".json",
+  ".yaml",
+  ".yml",
+  ".xml",
+  ".csv",
```

Comment:
bearish on CSV and XML

I think we should focus on "code" files unless explicitely asked by users?

Reply:
💯 with @spolu.

Reply:
Removed